rpc/codec/wire.pb/Makefile

```diff
@@ -15,14 +15,17 @@
 #
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-PROTOC      := ../../../../../../../bin/protoc
+# Ensure we only have one entry in GOPATH
+GOPATH := $(shell pwd)/../../../../../../..
+
+PROTOC      := $(GOPATH)/bin/protoc
 PROTOS      := $(wildcard *.proto)
 PROTO_GO    := $(PROTOS:.proto=.pb.go)
-GOGO_PROTOS := ../../../../github.com/gogo/protobuf/gogoproto/gogo.proto
+GOGO_PROTOS := $(GOPATH)/src/github.com/gogo/protobuf/gogoproto/gogo.proto
 
-PROTO_PATH := ../../../../../../:../../../../../../github.com/gogo/protobuf/protobuf:../../../../../../github.com/gogo/protobuf/gogoproto
+PROTO_PATH := $(GOPATH)/src/github.com/gogo/protobuf:$(GOPATH)/src/github.com/gogo/protobuf/protobuf
 
 all: $(PROTO_GO)
-
+.PHONY: $(PROTO_GO)
```

Comment:
not really, but this makefile doesn't know about the state of our gogoproto/protoc binaries, so this is safer